app/services/idv/phone_step.rb

```diff
@@ -146,5 +147,23 @@ def extra_analytics_attributes
         vendor: idv_result.except(:errors, :success),
       }
     end
+
+    def run_job(document_capture_session)
+      callback_url = Rails.application.routes.url_helpers.address_proof_result_url(
+        document_capture_session.result_id,
+      )
+
+      if Figaro.env.proofing_lambda_http_callback == 'true'
```

Comment:
I added `aws_lambda_proofing_enabled` in https://github.com/18F/identity-idp/pull/4264, do we need this other feature flag still?

Reply:
It needs a better name. This is the case where you don't want to use lambdas, but you do want an HTTP callback locally. Only case I think is in development, and probably not even on by default?

Reply:
I think for local development we should stick to the block form